app/components/mandaat/folded-fracties.js

```diff
@@ -17,6 +17,10 @@ export default class MandaatFoldedFractiesComponent extends Component {
     return this.args.bestuursperiode;
   }
 
+  get mandaat() {
+    return this.args.mandaat;
+  }
+
```

Comment:
I like it but  I can also see that is a little redundant for the times it is used